code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -623,6 +670,9 @@
 			use_obj = boots
 			check_slot = wearer.shoes
 		if("chest")
+			if (active)
```

Comment:
This seems too hardcoded, it should apply to any rig piece that encumbers the user, not just chest.

Reply:
It's not about encumbrance, but support. The chest braces the backpack against your body and provides it a sturdy base to operate its modules

Reply:
Which means it is hardcoded to this specific rig type where chests act as support. You plan to add rigs that can be deployed not just from back, and may not even have all types of parts.